ceph.spec.in

```diff
@@ -905,6 +905,7 @@ Requires: parted
 Requires: util-linux
 Requires: xfsprogs
 Requires: python%{python3_pkgversion}-setuptools
+Requires: python%{python3_pkgversion}-packaging
```

Comment:
@ktdreyer any objection about this? Is it fine to add that dependency?

Reply:
This is in RHEL 9's AppStream repositories, so it's already in RHEL 9 and it's fine to depend on it. :+1: